misc/python/materialize/parallel_workload/parallel_workload.py

```diff
@@ -415,6 +422,7 @@ def toggle_txn_wal(params: dict[str, str]) -> dict[str, str]:
             for action_class, count in counter.items()
         )
         print(f"  {error}: {text}")
+    assert failed < 50
```

Comment:
Is `50` just an arbitrary number?

Reply:
Yes, I mainly want to catch cases where most queries are uninteresting because of some syntax change.